i18n/zh-cn/docusaurus-plugin-content-docs/current/doc/gb28181.md

```diff
@@ -76,6 +76,53 @@ GB的Candidate定义和[WebRTC: Candidate](./webrtc.md#config-candidate)概念
 
 当然，如果网卡配置了客户端可以访问的地址，可以把`CANDIDATE`配置为`*`，让SRS自己发现。
 
+## External Sip
+
+目前SRS内置的SIP服务器仅实现了简单的`Register`、`Invite`指令，而要实现GB/T-28181的全部功能，势必会引入复杂的上层业务逻辑。因此，我们开发了一个独立的外置的SIP服务器。而SRS，只需开放几个简单的API接口，这样既保证了其媒体转发服务器的单一属性，又兼顾了与第三方SIP信令服务器对接的需求。
+
+仓库地址位于：[https://github.com/ossrs/srs-sip](https://github.com/ossrs/srs-sip)
+
+时序图如下所示：
+```mermaid
+sequenceDiagram
+    Player ->> SRS-SIP : 1. Play Request(with id)
+    SRS-SIP ->>  SRS : 2. Publish Request(with ssrc and id)
+    SRS ->> SRS-SIP : 3. Response(with port)
+    SRS-SIP ->>  Device : 4. Invite(with port)
+    Device ->> SRS-SIP : 5. 200 OK
+    SRS-SIP ->> Player : 6. 200 OK(with url)
+    Device -->> SRS : Media Stream
+    Player ->> SRS : 7. Play
+    SRS -->> Player : Media Stream
+    Player ->> SRS-SIP : 8. Stop Request
+    SRS-SIP ->> SRS : 9. Unpublish Request
+    SRS-SIP ->> Device : 10. Bye
+```
```

Comment:
About this sequence diagram, there is a camera device registration interaction before step 1 - Play Request, missing this step will make me confused at first. What's more, this diagram is for `srs-sip`, every primary steps of sip should be included.

The internal sip would invite the device to push stream immediately, while this external sip would only invite if there are players, why not let this sip also support auto invite mode.

About the Player, How about let player interact with the SRS, let the SRS ask `SRS-SIP` to invite device if the device is not yet published its steam, add an api to query registered device ids in `SRS-SIP`.

`Player` -- query devices id --> `SRS-SIP`.
`Player` -- play media streams with device id -> `SRS` -- if no such streams request device invite --> `SRS-SIP` (player retry this steps if still no media streams)

Reply:
1. I added a registration process.
2. The `auto invite mode` will be implemented in future versions.
3. I disagree about this point. This will lead to coupling between SRS and business. The current process is clear: first request from SRS-SIP, and get the url, then pull stream from SRS.

Reply:
OK, I knows more GB-28181 now.